pac4j-oidc/src/main/java/org/pac4j/oidc/profile/OidcProfile.java

```diff
@@ -55,6 +56,14 @@ public JWT getIdToken() throws ParseException {
         return null;
     }
 
+    public String getRefreshTokenString() {
```

Comment:
Yes, I think refresh tokens are like access tokens: opaque string, by opposition to the id token.